### PR TITLE
Fix logbook username not appearing consistently

### LINKED ISSUE #6229: user of a logbook message not consistently shown
## Checklist

- [x] I have updated to the latest available Home Assistant version.
- [x] I have cleared the cache of my browser.
- [x] I have tried a different browser to see if it is related to my browser.

## The problem

The new user field of a logbook message is not consistently shown.  Refreshing the logbook view can remove the (user) from the log messages.  Additional refreshes can change which message have the user populated.  Scrolling the messages with the missing user off the screen and back can restore the value.  See: https://imgur.com/a4YpNW3 

## Expected behavior

(User) should always be populated when the logbook messages are displayed.
## Steps to reproduce

1. Start with a logbook screen with action messages and a populated (user) 
2. Click refresh in upper right
3. Some/all action messages should have 'lost' the (user) info
4. Refreshing again can change which messages have the (user) data
5. Scrolling down the logbook list and back up will repopulate the user info for the messages that were out of view. It's not always all of them, it seems like the further you scroll the more that are off screen will be corrected

## Environment

- Home Assistant release with the issue: 0.112.0b0
- Last working Home Assistant release (if known): none (new feature)
- Browser and browser version: (Chrome 84.0.4147.68 (Official Build) beta (64-bit)) and Version 85.0.4181.9 (Official Build) canary (32-bit)
- Operating system: Windows 10

## State of relevant entities

```yaml

```

## Problem-relevant configuration

```yaml

```

## Javascript errors shown in your browser console/inspector

```txt
A cookie associated with a cross-site resource at http://home-assistant.io/ was set without the `SameSite` attribute. A future release of Chrome will only deliver cookies with cross-site requests if they are set with `SameSite=None` and `Secure`. You can review cookies in developer tools under Application>Storage>Cookies and see more details at https://www.chromestatus.com/feature/5088147346030592 and https://www.chromestatus.com/feature/5633521622188032.
integrations:1 

A cookie associated with a cross-site resource at https://home-assistant.io/ was set without the `SameSite` attribute. A future release of Chrome will only deliver cookies with cross-site requests if they are set with `SameSite=None` and `Secure`. You can review cookies in developer tools under Application>Storage>Cookies and see more details at https://www.chromestatus.com/feature/5088147346030592 and https://www.chromestatus.com/feature/5633521622188032.
:8123/developer-tools/state:1 

Autofocus processing was blocked because a document already has a focused element.

DevTools failed to load SourceMap: Could not load content for chrome-extension://hdokiejnpimakedhajhdlcegeplioahd/sourcemaps/onloadwff.js.map: HTTP error: status code 404, net::ERR_UNKNOWN_URL_SCHEME
```

### PR DESCRIPTION
**Pull request by bdraco**
##  Proposed change

Fix logbook username not appearing consistently
Only lookup the user once when the panel is loaded.

## Type of change

- [ ] Dependency upgrade
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #6229
- This PR is related to issue:
- Link to documentation pull request:

## Checklist

- [x] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/panels/logbook/ha-logbook.ts [hunk 1]
```diff
@@ -23,7 +22,7 @@ class HaLogbook extends LitElement {
   @property() public hass!: HomeAssistant;
 
   @property({ attribute: false })
-  private _userid_to_name = {};
+  private userid_to_name = {};
```
Comment (balloob) [inline src/panels/logbook/ha-logbook.ts:25]: Make it public and use camel case.

**Review by balloob — APPROVED** [review]
I think what's happening is that the logbook is not re-rendering if the users come in after the data has been fetched. So scrolling renders new rows and that works. Because of the speed improvements, it can now happen that logbook is returned faster.

We might want to consider showing a spinner until both users and logbook are loaded. I can look at that soon

File: src/panels/logbook/ha-panel-logbook.ts [hunk 2]
```diff
@@ -184,6 +191,15 @@ export class HaPanelLogbook extends LitElement {
     }
   }
 
+  private async _fetchUsers() {
```
Comment (balloob) [inline src/panels/logbook/ha-panel-logbook.ts:194]: We need to compute this in `_getData` and only set loading=false once users are resolved. We can do this by storing the promise that is returned by `this._fetchUsers` and using `await Promise.all([])`. It's late here now but I can fix that tomorrow 👍

Comment (bdraco) [reply src/panels/logbook/ha-panel-logbook.ts:194]: Any assistance here would be appreciated as I'm still getting my feet wet on the frontend.  Thanks @balloob

**Review by balloob — APPROVED** [review]

**Pull request merged**